resources/views/dashboard/controllers/staff.blade.php

```diff
@@ -180,4 +180,4 @@
     </div>
 </div>
 
-@endsection
```

Comment:
What happened on line 183? There shouldn't have been any changes here. Can't really tell what happened from the change history.

Reply:
I'm honestly not sure what has happened if I try to redo it and change just the text, there are no changes to push.

Reply:
Ok, no problem. Just wanted to ask.